### DISCUSSION TITLE
README.md 显示不正常  经常oops 404

### DISCUSSION THREAD

**Discussion by hi-pengyu** [spencerwooo/onedrive-vercel-index#554] [Q&A] [answered]
在浏览器没有缓存下，readme.md显示时候经常404，需要刷新一两次就能显示正常，这是啥情况

**Comment by spencerwooo** [comment, marked as answer]
是最新部署版本吗？老版本 7df0453269af1f838556c10b7fbb06188b49691a 之前没有对 README 文件做 proxy，在那个 commit 之后加入了对 README 内容的 Vercel 转发和缓存功能 - 理论上其实应该加载更稳定。
